docs/introduction.rst

```diff
@@ -66,34 +50,52 @@ illustrative purposes, and converts the reduced-variables model to QUBO formulat
     >>> bqm_no3_qubo.vartype
     <Vartype.BINARY: frozenset([0, 1])>
 
-
 Samplers and Composites
 -----------------------
 
-*Samplers* are processes that sample from low energy states of a problem’s objective function.
-A binary quadratic :term:`model` (BQM) :term:`sampler` samples from low energy states in
-models such as those defined by an Ising equation or a QUBO problem and returns an
-iterable of samples, in order of increasing energy. A dimod
+:term:`Sampler`\ s are  processes that sample from low-energy states of a problem’s objective function.
```

Comment:
The trailing backslashes raise a lot of errors for my (and assumedly the customer's) linter. It would be good if we could avoid them whenever possible.

Reply:
I do try to minimize use. Consider making a PR to improve the linter here: https://github.com/PyCQA/pylint/